content/cpp/concepts/vectors/vectors.md

```diff
@@ -91,6 +91,24 @@ what():  vector::_M_range_check: __n (which is 100) >= this->size() (which is 3)
 Aborted (core dumped)
 ```
 
+### `.front()`
+
+The `.front()` returns a reference to the first element of the vector:
+
+```cpp
+// The first element of a vector
+std::cout << arr.front();
```

Comment:
```suggestion
std::cout << order.front();
```

Reply:
Since this is adding to the section "getting information" which uses a vector declared in the first example throughout all the code blocks, it should be carried over here as well